.github/workflows/scripts/win_build.ps1

```diff
@@ -29,15 +26,15 @@ if (-not (Test-Path $libsDir)) {
     New-Item -ItemType Directory -Path $libsDir
 }
 Push-Location $libsDir
-WriteInfo("Download and extract LLVM")
 if (-not (Test-Path "taichi_llvm")) {
+    WriteInfo("Download and extract LLVM")
```

Comment:
Don't need to change anything here, tested on the github machines, 7z is preinstalled.